UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -207,16 +209,17 @@ public void RestartRound()
 	{
 		if (CustomNetworkManager.Instance._isServer)
 		{
-			if (MapRotationCount <= 10)
-			{
-				CustomNetworkManager.Instance.ServerChangeScene ("Assets/scenes/OutpostDeathmatch.unity");
+			if (MapRotationCount < 10) {
+				CustomNetworkManager.Instance.ServerChangeScene (Scene[1]);
+				MapRotationCount++;
 			} 
-			else if (MapRotationCount > 10) 
+			else if (MapRotationCount >= 10) 
 			{
-				CustomNetworkManager.Instance.ServerChangeScene ("Assets/scenes/Flashlight Deathmatch.unity");
+				CustomNetworkManager.Instance.ServerChangeScene (Scene[2]);
+				MapRotationCount++;
 				if (MapRotationCount == 20) 
 				{
-					MapRotationCount = 19;
+					MapRotationCount = 0;
 				}
 			}
 		}
```

Comment:
I generally would prefer a more generic approach that can deal with a variable size of scenes. Not sure, if that was a requirement of the issue, though.